### PR TITLE
CLI and better checks

### PR DESCRIPTION
**Pull request by lachmanfrantisek**

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
The target resolution needs master version of conu ([this pull request](https://github.com/fedora-modularity/conu/pull/168)).

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='colin',
+    version='0.1',
+    packages=find_packages(),
+    install_requires=[
+        'Click',
```
Comment (TomasTomecek) [inline setup.py]: someone doesn't like argparse

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='colin',
+    version='0.1',
```
Comment (TomasTomecek) [inline setup.py]: [sem-ver please](https://semver.org/)

File: colin/core/target.py [hunk 3]
```diff
@@ -25,7 +26,7 @@ def _get_target_instance(target_name):
         :param target_name: str
         :return: Container/Image
         """
-        with DockerBackend() as backend:
+        with DockerBackend(logging_level=logging.NOTSET) as backend:
```
Comment (TomasTomecek) [inline colin/core/target.py]: does this mean "no logs"?

File: colin/cli/colin.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+@click.option('--config', '-c', type=click.STRING,
+              help="Configuration name.")
+def cli(target, config):
+    results = run(name_of_target=target,
+                  config_name=config)
+
+    for r in results.results:
+        click.secho(str(r), err=r.ok, fg='green' if r.ok else 'red')
+        if not r.ok:
+            click.secho("   -> {}\n"
```
Comment (TomasTomecek) [inline colin/cli/colin.py]: good enough for PoC, [we need to make this better in future](https://github.com/user-cont/colin/issues/14)

File: colin/cli/colin.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+from ..core.colin import run
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('target', type=click.STRING)
+@click.option('--config', '-c', type=click.STRING,
+              help="Configuration name.")
+def cli(target, config):
```
Comment (TomasTomecek) [inline colin/cli/colin.py]: RFE https://github.com/user-cont/colin/issues/15: does not block this PR

File: setup.py (hunk 1 above)
Comment (lachmanfrantisek) [reply setup.py]: Yes..;-) Is it a big problem?

File: colin/core/target.py (hunk 3 above)
Comment (lachmanfrantisek) [reply colin/core/target.py]: I just need to turn off the `conu` logs. Is there any better approach to do this?

File: setup.py (hunk 1 above)
Comment (TomasTomecek) [reply setup.py]: I don't think so, let's wait what @phracek would think.

File: colin/core/target.py (hunk 3 above)
Comment (TomasTomecek) [reply colin/core/target.py]: don't know; if this works, we're good, if not, we may have to fix that in conu (something like: "don't setup logging")

**Pull request merged**